src/extra/gis.jl

```diff
@@ -63,30 +53,46 @@ function gistable(fname; layer, numtype, kwargs...)
     return GJS.read(fname; numbertype=numtype, kwargs...)
   elseif endswith(fname, ".parquet")
     return GPQ.read(fname; kwargs...)
+  elseif endswith(fname, ".gpkg")
+    return gpkgtable(fname; layer)
   else # fallback to GDAL
     data = AG.read(fname; kwargs...)
     return AG.getlayer(data, layer - 1)
   end
 end
 
 # helper function to convert Tables.jl table to GeoTable
-function asgeotable(table)
-  crs = GI.crs(table)
-  cols = Tables.columns(table)
+function asgeotable(rawtable)
+  # table of attributes and column of geometries
+  cols = Tables.columns(rawtable)
   names = Tables.columnnames(cols)
   gcol = geomcolumn(names)
   vars = setdiff(names, [gcol])
-  etable = isempty(vars) ? nothing : namedtuple(vars, cols)
+  table = isempty(vars) ? nothing : namedtuple(vars, cols)
   geoms = Tables.getcolumn(cols, gcol)
-  # subset for missing geoms
+
+  # identify rows with missing geometries
   miss = findall(g -> ismissing(g) || isnothing(g), geoms)
   if !isempty(miss)
     @warn "Dropping $(length(miss)) rows with missing geometries. Please use `GeoIO.loadvalues(fname; rows=:invalid)` to load their values."
   end
   valid = setdiff(1:length(geoms), miss)
-  domain = geom2meshes.(geoms[valid], Ref(crs))
-  etable = isnothing(etable) || isempty(miss) ? etable : Tables.subset(etable, valid)
-  georef(etable, domain)
+
+  # subset table and geometries
+  stable = isnothing(table) || isempty(miss) ? table : Tables.subset(table, valid)
+  sgeoms = geoms[valid]
```

Comment:
You didn't include my last change in this line. Please make sure to include all the work from the long iterations we had in the other PR.